phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -3661,7 +3681,7 @@ private MutationCode processMutationResult(String schemaName, String tableName,
             .setSchemaName(schemaName).setTableName(tableName).setFamilyName(familyName).setColumnName(columnName).setMessage(msg).build().buildException();
         case UNALLOWED_SCHEMA_MUTATION:
             throw new SQLExceptionInfo.Builder(
-                    SQLExceptionCode.CANNOT_SET_OR_ALTER_PHOENIX_TTL)
+                    SQLExceptionCode.CANNOT_SET_OR_ALTER_TTL)
```

Comment:
Just curious, why is `UNALLOWED_SCHEMA_MUTATION` unconditionally mapped to `CANNOT_SET_OR_ALTER_TTL`?

Reply:
We are using that Mutation Code only in one place when we validate PhoenixTTLAttribiteSettingForView from our previous View TTL design :- https://github.com/apache/phoenix/blob/c4b1af2cb4ceacfb5453285c4b61c49e6954cdb6/phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java#L3123
I agree this can be renamed to something related to TTL.